packages/status-core/src/utils/public-key-to-color-hash.test.ts

```diff
@@ -0,0 +1,63 @@
+import {
```

Comment:
Can be tested locally with `yarn run jest packages/status-core/src/utils/public-key-to-color-hash.test.ts`.

<img width="588" alt="image" src="https://user-images.githubusercontent.com/13265126/169716094-54897657-92f2-48fa-bd5e-506db88a713c.png">